rust/automerge/src/op_set2/change.rs

```diff
@@ -41,16 +41,35 @@ pub(crate) fn build_change<T, G>(
     graph: &G,
     actors: &[ActorId],
 ) -> Change<'static, Verified>
+where
+    T: AsChangeOp,
+    G: GetHash,
+{
+    let mut mapper = ActorMapper::new(actors);
+    build_change_inner(ops, meta, graph, &mut mapper)
+}
+
+#[inline(never)]
```

Comment:
Does the `inline(never)` make a big difference?

Reply:
Yeah I would be nervous about leaving them in until we have benchmarks running in CI on all major architectures.